docs/reference/data-streams/logs.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[logs-data-stream]]
+== Logs data stream
+
+preview::[Logs data streams and the logs index mode are in tech preview and may be changed or removed in the future. Don't use logs data streams or logs index mode in production.]
+
+A logs data stream is a data stream type that stores log data more efficiently.
+
+In benchmarks, log data stored in a logs data stream used ~2.5 times less disk space than a regular data
+stream. The exact impact will vary depending on your data set.
+
+The following features are enabled in a logs data stream:
+
+* <<synthetic-source,Synthetic source>>, which omits storing the `_source` field. When the document source is requested, it is synthesized from document fields upon retrieval.
+
+* Index sorting. This yields a lower storage footprint. By default indices are sorted by `host.name` and `@timestamp` fields at index time, which can be overwritten.
+
+* More space efficient compression for fields with <<doc-values,`doc_values`>> enabled.
+
+The new logs data stream should be used for all data streams that store logs, because of the storage benefits that comes with it.
```

Comment:
since we're telling people not to use this in prod above, I'd probably exclude this for now. 

was hoping to see a dedicated "when to use" section like [this one](https://www.elastic.co/guide/en/elasticsearch/reference/current/tsds.html#when-to-use-tsds), that also maybe pointed out shortcomings or gotchas

Reply:
I think we should then address this in a follow up. I will remove it for now.